src/main/kotlin/de/livepoll/api/entity/dto/MultipleChoiceItemDtoOut.kt

```diff
@@ -6,5 +6,5 @@ class MultipleChoiceItemDtoOut(
         question: String,
         position: Int,
         type: String,
-        val answers: List<MultipleChoiceItemAnswerDtoOut>
+        val selectionOptions: List<MultipleChoiceItemAnswerDtoOut>
```

Comment:
These are supposed to be answers including selection options. Please undo this commit.